spec/helpers/i18n_helper_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+RSpec.describe I18nHelper do
+  describe '#date_eras_gregorian' do
+    context 'before christ' do
+      it 'converts gregorian before christ to correct format' do
```

Comment:
Let's refer to "CE" or "BCE" instead of "before christ" and "after christ" in the specs.